apps/communications/contacts/services/contacts.js

```diff
@@ -162,6 +162,17 @@
         }
       };
       cursor.onerror = onErrorCB;
+    },
+
+    getCount: function(callback) {
```

Comment:
I think you don't need this method. We have another one called `isEmpty` that probably can be used in here (`getCount ` is used with an `IF (count > 1)...`)